### PR TITLE
New runtime blurb message.

### PR DESCRIPTION
**Pull request by hjoliver**
A bit of fluff.

Old:
old-blurb

New:
new-blurb

### CODE REVIEW

**Comment by hjoliver** [comment]
@matthewrmshin - please review.

File: lib/cylc/run.py [hunk 1]
```diff
@@ -27,29 +28,34 @@
...
+This program comes with ABSOLUTELY NO WARRANTY.   It is
+free software; you are welcome to redistribute it under
+certain conditions: "cylc warranty", "cylc conditions".
+ 
+  """ % CYLC_VERSION
+    
+    logo_lines = logo.splitlines()
+    license_lines = license.splitlines()
+    lmax = max(len(line) for line in license_lines)
+    for i in range(0,len(logo_lines)):
```
Comment (matthewrmshin) [inline lib/cylc/run.py]: `0,` not necessary here?

**Comment by matthewrmshin** [comment]
Looks OK to me.

@benfitzpatrick please sanity check.

**Comment by benfitzpatrick** [comment]
Looks nice.

**Pull request merged**